tests/Forms/EditFormFactoryTest.php

```diff
@@ -21,4 +24,42 @@ public function testFormFieldsHaveNamespaces()
 
         $this->assertNotNull($fields->dataFieldByName('PageElements_' . $record->ID . '_Title'));
     }
+
+    public function testNamespaceFields(): void
+    {
+        $record = $this->objFromFixture(ElementContent::class, 'content_block');
+        $factory = new EditFormFactory();
+        $fields = new FieldList([
+            new TextField('FieldOne'),
+            new LiteralField('IgnoredField', ''),
```

Comment:
`LiteralField` is ignored because it's not a "data field" so it doesn't affect form submission